### ISSUE TITLE
If leave is claimed, day start and end times are still filled.

### ISSUE THREAD

**Issue by lukecharde** [patik/lats-helper#2]
User Story: If time is charged to leave (vacation, personal, etc.) that day should not be autofilled so that the user does not need to manually delete the auto-filled entries.

![my_timesheet](https://cloud.githubusercontent.com/assets/782614/9040680/aad9bd76-39d1-11e5-8ad7-6fdd2d9a5358.png)

**Comment by patik** [comment]
I've been meaning to add an autofill mechanism where you could enter your in/out/lunch and then automatically fill in a leave category with the remaining number of hours needed to make a full day. I get the sense that it ties in with your request but I'm not sure exactly how.

**Comment by lukecharde** [comment]
Agreed... related.  In this case, I don't fill out any time, only vacation/sick hrs (out an entire day), so then have to go in and manually delete the auto-fill.

**Comment by lukecharde** [comment]
Another screenshot...  in this case there was holiday and a sick day - and after auto-filling I had to delete entries for those days.  It's the full days of no work that are the issue.

![timesheet](https://cloud.githubusercontent.com/assets/782614/12690352/c79be81e-c6b0-11e5-9703-c5b3dc17066e.jpg)

A day with a dr's appointment where I entered alternative times was skipped (appropriately) during the auto-fill.

![timesheet](https://cloud.githubusercontent.com/assets/782614/12690376/f731f258-c6b0-11e5-96d7-48614691a725.jpg)
